### PR TITLE
docs(docs-infra): move back all xterm code to the fw repo

### PR DESCRIPTION
**Pull request by JeanMeche**
Previously, the `@angular/docs` package had `xterm` as dependency but wasn't referencing it. It only had dedicated styles. This commit moves back the styles to the FW repo as it is only used here.

* Pending angular/dev-infra#2043

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -183,6 +184,7 @@
     "@types/inquirer": "^9.0.3",
     "@types/jsdom": "^21.1.5",
     "@typescript/vfs": "^1.5.0",
+    "@xterm/xterm": "^5.4.0",
```
Comment (devversion) [inline package.json:187]: Is there a reason the other dependency is in `dependencies` and this not?

File: adev/angular.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
             "tsConfig": "tsconfig.app.json",
             "inlineStyleLanguage": "scss",
             "assets": ["src/favicon.ico", "src/robots.txt", "src/assets"],
-            "styles": ["@angular/docs/styles/global-styles.scss"],
+            "styles": ["@angular/docs/styles/global-styles.scss", ".src/local-styles.scss"],
```
Comment (devversion) [inline adev/angular.json:31]: Is this dot wrong?

**Review by devversion — APPROVED** [review]
LGTM, assuming the Sass is the same? (if not, we should get additional review)

**Review by devversion — APPROVED** [review]
LGTM

**Comment by thePunderWoman** [comment]
This PR was merged into the repository by commit 7928e11a0e5a36b8c2b93ac9b2dad196e120d7fe.

**Pull request closed without merging**